edb/server/bootstrap.py

```diff
@@ -358,6 +358,7 @@ def _process_delta(delta, schema):
 async def _populate_data(std_schema, schema, conn):
     script = '''
         INSERT stdgraphql::Query;
+        INSERT stdgraphql::Mutation;
```

Comment:
AFAIR, this was causing a crash in bootstrap due to a bug I fixed, but you put the fix into a subsequent commit, please rearrange correctly.  Alternatively, just squash the commits.

Reply:
OK, moved the fix to the earlier commit. The graphql tests pass for that commit now.